loky/backend/spawn.py

```diff
@@ -240,3 +242,68 @@ def _fixup_main_from_path(main_path):
                                   run_name="__mp_main__")
     main_module.__dict__.update(main_content)
     sys.modules['__main__'] = sys.modules['__mp_main__'] = main_module
+
+
+def main(fd, process_name):
+    fd = int(fd)
+    if sys.platform == "win32":
+        fd = msvcrt.open_osfhandle(fd, os.O_RDONLY)
+
+    exitcode = 1
+    try:
+        with os.fdopen(fd, 'rb') as from_parent:
+            process.current_process()._inheriting = True
+            try:
+                prep_data = pickle.load(from_parent)
+                prepare(prep_data)
+                process_obj = pickle.load(from_parent)
+            finally:
+                del process.current_process()._inheriting
+
+        exitcode = process_obj._bootstrap()
+    except Exception:
+        print('\n\n' + '-' * 80)
+        print(f'{process_name} failed with traceback: ')
+        print('-' * 80)
+        import traceback
+        print(traceback.format_exc())
+        print('\n' + '-' * 80)
+    finally:
+        if from_parent is not None:
+            from_parent.close()
+
+        sys.exit(exitcode)
+
+
+def get_command_line(main_prog=main, **kwargs):
+    '''
+    Returns prefix of command line used for spawning a child process
+    '''
+
+    if getattr(sys, 'frozen', False):
+        list_kwargs = [f'{k}={v}' for k, v in kwargs.items()]
+        argv = [
+            sys.executable, '--multiprocessing-fork', main_prog.__module__,
+            *list_kwargs
+        ]
+    else:
+        list_kwargs = [f'{k}="{v}"' for k, v in kwargs.items()]
+        prog = (
+            f'from {main_prog.__module__} import main; '
+            f'main({", ".join(list_kwargs)})'
+        )
+        opts = util._args_from_interpreter_flags()
+        argv = [get_executable(), *opts, '-c', prog]
+    return argv
+
+
+def freeze_support():
```

Comment:
If this works, this need a proper doc string and some comment for later readability.

Reply:
```suggestion
def freeze_support():
    """Run code for the child workers when necessary.
    
    This helper allows the frozen executable to call the code for the child
    workers when not in the main process.
    It should be called right after the beginning of the programme, to
    avoid recursive process spawning.
    """
```